classes/Tools.php

```diff
@@ -3733,9 +3733,7 @@ public static function spreadAmount($amount, $precision, &$rows, $column)
             return;
         }
 
-        $sort_function = create_function('$a, $b', "return \$b['$column'] > \$a['$column'] ? 1 : -1;");
-
-        uasort($rows, $sort_function);
+        uasort($rows, function ($a, $b) { return $b['$column'] > $a['$column'] ? 1 : -1; });
```

Comment:
Pass `$column` var in anonymous function to be able to use it. 
Remove single inverted comma from variable index `$b[$column] `

should be like
`uasort($rows, function ($a, $b) use ($column) { return $b[$column] > $a[$column] ? 1 : -1; });`

Reply:
@shreesh-webkul done.